### DISCUSSION TITLE
Survey on globally distributed software engineering

### DISCUSSION THREAD

**Discussion by AsWali** [eclipse-theia/theia#9454] [General] [answered]
Hey there. 

We are doing a survey on globally distributed software engineering focused on open source projects. If you have some spare time we would really appreciate your response. The form takes about 10 minutes. Thanks in advance!
Survey can be found [here](link).

**Comment by vince-fugnitto** [comment, marked as answer]
As other repositories have pointed out, there is no clear indication of who the survey is for, and for what purpose so marking it as spam.
